kafka-ui-api/pom.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Why changes here? Is there a way to not have this changes in this file?

Reply:
We need the addition of this dep so tings build properly
```
<dependency>
            <groupId>com.provectus</groupId>
            <artifactId>kafka-ui-proto-gen</artifactId>
            <version>${project.version}</version>
        </dependency>
```